### DISCUSSION TITLE
Public Facing Dashboard

### DISCUSSION THREAD

**Discussion by JMadisonTP** [btcpayserver/btcpayserver#5436] [Ideas / Feature Requests]
I was talking with someone and he had a great idea.  We have the internal dashboard as a admin to view all stores but also store owners to get a high overview of tx, balances, products sold, and other useful stuff. 

The idea: 

What if there was a way to pull some of that information, via api or something similar to the 'view' button inside the PoS Store.
And kinda show the status health of a store that accepts LN, how many txs, products sold (popular), kinda like a 10k foot look down, type information. that store owners can display to show a real impact taking btc payments.

Example, 
what if like you on-boarded a small community of merchants, that you were running stores for, but want to show the public, how poplar bitcoin payments in your community is and the impact it is having. Would be really cool information that merchants and users of bitcoin can see and appreciate. 

Imagine a place like Bitcoin Park, Pleb Lab, Bitcoin Beach etc, if they are using btcpay - display some kind of public facing dashboard with some neat metrics.  I think this would be a big boost for merchants and btcpay alike. 

Thank you for your time.

**Comment by dstrukt** [comment]
Interesting idea, will think on this some more - might follow up with questions as well!
